src/components/Navbar/navbar.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import '../../css/navbar.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faHome,
+  faClipboard,
+  faSignOutAlt,
+  faCalendarCheck,
+  faCar,
+} from '@fortawesome/free-solid-svg-icons';
+import { openForm } from '../../redux/car_list/addNewCarFormSlice';
+
+const Navbar = (props) => {
+  const dispatch = useDispatch();
+  const { Page } = props;
+  const openFormHandle = () => {
+    dispatch(openForm());
+  };
+  return (
+    <div className="container-fluid">
+      <div className="row flex-nowrap">
+        <div className="col-auto col-md-3 col-xl-2 px-sm-2 px-0 bg-white">
+          <div className="border-right d-flex flex-column align-items-center align-items-sm-start px-0 pt-2 text-white min-vh-100">
+            <NavLink
+              exact
+              to="/"
```

Comment:
On line 26 there is a right margin for the left column which is the navbar. Could you please remove only the right padding? You could replace  the `px-sm-2` class for `ps-sm-2` so only the left padding is modified.